services/configuration/src/config_api.rs

```diff
@@ -261,8 +263,8 @@ where
         let ch = self.channel.clone();
         ch.send(Box::new(config_propose))?;
         let res = ApiResponseProposePost {
-            tx_hash: tx_hash,
-            cfg_hash: cfg_hash,
+            tx_hash,
+            cfg_hash,
```

Comment:
FWIW, IDEA can apply `foo: foo -> foo` fix automatically to the whole project :)